bitbots_body_behavior/bitbots_body_behavior/actions/go_to_block_position.py

```diff
@@ -40,9 +40,9 @@ def perform(self, reevaluate=False):
         pose_msg.header.stamp = self.blackboard.node.get_clock().now().to_msg()
         pose_msg.header.frame_id = self.blackboard.map_frame
 
-        pose_msg.pose.position.x = -(self.blackboard.world_model.field_length / 2) + self.block_position_goal_offset
-        pose_msg.pose.position.y = self._stay_in_front_of_goal(goalie_y)
-        pose_msg.pose.orientation.w = 1
+        pose_msg.pose.position.x = float(-(self.blackboard.world_model.field_length / 2) + self.block_position_goal_offset)
```

Comment:
As the error is concerned with the quaternion and this is the position and it includes a float division the float is not nessesary, but it is not really hurting ether.